src/uct/ib/rc/base/rc_iface.c

```diff
@@ -178,6 +178,13 @@ ucs_status_t uct_rc_iface_query(uct_rc_iface_t *iface,
     }
 #endif
 
+    if (dev->flags & UCT_IB_DEVICE_FLAG_PCI_ATOMICS) {
+        iface_attr->cap.atomic32.op_flags = 0;
+        iface_attr->cap.atomic32.fop_flags = 0;
+        iface_attr->cap.atomic64.op_flags = 0;
```

Comment:
why are all atomics disabled now? is it going to be fixed in next PR?

Reply:
yes